src/sentry/api/endpoints/organization_integration_repos.py

```diff
@@ -44,6 +45,11 @@ def get(
             context = {"repos": []}
             return self.respond(context)
 
+        installed_repos = Repository.objects.filter(integration_id=integration.id).exclude(
+            status=ObjectStatus.HIDDEN
+        )
+        external_names = {hidden_repo.name for hidden_repo in installed_repos}
```

Comment:
I'm not really sure what `external_names` here means, what does this list represent and is there a name that can better represent that?

Reply:
`Repository` names in sentry are formatted like `getsentry/sentry`. this is how we refer to them and i'm not sure there's a better way to name the variable. maybe `repo_names`, but it's still not very descriptive

Reply:
cool if its consistent thats all g